### DISCUSSION TITLE
Is it possible to only forward outgoing mail?

### DISCUSSION THREAD

**Discussion by madhavgharmalkar** [haraka/Haraka#3231] [Q&A] [answered]
Hello,

I was wondering it is possible to forward only outgoing mail with the `queue/smtp_forward` plugin? I'd like to receive mail and forward it to my IMAP server via LMTP (`queue/lmtp`)  and forward my outgoing mail to SES / Sendgrid etc.

Is this possible?

**Comment by msimerson** [comment, marked as answer]
Yes, it is possible. It's also possible to be even more specific. Plugins can set `queue.wants` to whichever queue plugin they wish to deliver a message and `queue.next_hop` to further direct that queue plugin.
